cmd/antrea-agent/options_windows_test.go

```diff
@@ -0,0 +1,90 @@
+// +build windows
```

Comment:
quick note that I'm not sure this test will be run in CI - it's a pretty straightforward change but at the moment I don't think we run Windows unit tests in Github workflows

Reply:
Did not know this, but I saw several other Windows only unit tests too.

Reply:
Odds are, if you add `windows` to the `Unit test` job, it will just work

Reply:
Added.

Reply:
I found I need to mark some tests as Linux only, and fix some Windows tests. If you agree, I would create another PR to enable Windows unit test on github.